tests/unit_tests/test_rating_service.py

```diff
@@ -343,3 +343,20 @@ async def test_game_rating_error_handled(rating_service, game_info, bad_game_inf
     service._logger.warning.assert_called()
     # second game: results have been saved.
     service._persist_rating_changes.assert_called_once()
+
+
+async def test_game_update_empty_resultset(rating_service):
+    service = rating_service
+    service._persist_rating_changes = CoroutineMock()
+    game_id = 2
+    player_id = 1
+    rating_type = RatingType.GLOBAL
+    rating_type_id = service._rating_type_ids[RatingType.GLOBAL]
+    old_ratings = {player_id: Rating(1000, 500)}
+    after_mean = 1234
```

Comment:
Not sure that you need a separate variable here, you don't have one for before_mean and this isn't used anywhere else in the test.